### PR TITLE
Update Booleans.java

### PR DESCRIPTION
**Pull request by Lakhan-Nad**
updated doc for compare function

### CODE REVIEW

**Comment by ljacqu** [comment]
I don‘t think that‘s correct

File: guava/src/com/google/common/primitives/Booleans.java [hunk 1]
```diff
@@ -122,7 +122,7 @@ public static int hashCode(boolean value) {
    * @param a the first {@code boolean} to compare
    * @param b the second {@code boolean} to compare
    * @return a positive number if only {@code a} is {@code true}, a negative number if only {@code
-   *     b} is true, or zero if {@code a == b}
+   *     b} is {@code false}, or zero if {@code a == b}
```
Comment (nick-someone) [inline guava/src/com/google/common/primitives/Booleans.java:125]: I appreciate the edit to add code formatting, but this should be `{@code true}`

Comment (Lakhan-Nad) [reply guava/src/com/google/common/primitives/Booleans.java:125]: okay will change that. Thankyou

**Comment by rohitnandi12** [comment]
@kluever This could be closed too. 
PS: Is there anything I can work on? I am new to open source contribution.

**Pull request closed without merging**